### ISSUE TITLE
Output file naming convention

### ISSUE THREAD

**Issue by rjarvinen** [iljah/pamhd#2]
It would be useful to name PAMHD output files such that they are correctly ordered in time in Linux, VisIt, etc. Now the name includes simulation time in scientific notation, which is somewhat cumbersome to deal with.

**I suggest replacing the time parameter in the file name with an integer giving the number of time steps since the start**. With appropriate amount of padding zeros of course.

For example:

mhd_staggered_7.205e+03_s.dc
->
mhd_staggered_00000698.dc

**Comment by iljah** [comment]
Makes sense, file header includes simulation time so information wouldn't be lost.

**Comment by iljah** [comment]
Fixed for staggered test and mhd2gnuplot converter by https://github.com/iljah/pamhd/commit/b7e079f4fa42d84907fdc654eaef67f316621d1d

**Comment by iljah** [comment]
mhd2vtk_staggered.exe works fine for me but I'm using amr branch

**Comment by iljah** [comment]
I removed staggered branch since staggered solver development is in amr branch now. If there are further problems there please open another issue.
